includes/admin/class-wcmypa-admin.php

```diff
@@ -1172,11 +1177,26 @@ public function renderBarcodes(WC_Order $order): void
         $exportMode = WCMYPA()->setting_collection->getByName(WCMYPA_Settings::SETTING_EXPORT_MODE);
 
         if (WCMP_Settings_Data::EXPORT_MODE_PPS === $exportMode) {
-            $orderId = WCX_Order::get_id($order);
-            $metaPps = get_post_meta($orderId, self::META_PPS);
+            $orderId   = WCX_Order::get_id($order);
+            $metaPps   = get_post_meta($orderId, self::META_PPS);
 
             if ($metaPps) {
-                echo esc_html(sprintf(__('export_hint_how_many_times', 'woocommerce-myparcel'), count($metaPps)));
+                $lastOrder = end($metaPps);
+
+                if (isset($lastOrder[self::META_PPS_UUID])) {
+
+                    echo $lastOrder[self::META_TRACK_TRACE] ? sprintf(
+                        '<a href="%s">%s</a>',
+                        TrackTraceUrl::create(
+                            $lastOrder[self::META_TRACK_TRACE],
+                            $order->get_shipping_postcode(),
+                            $order->get_shipping_country()
+                        ),
+                        $lastOrder[self::META_TRACK_TRACE]
+                    ) : esc_html(__('Concept created but not printed.', 'woocommerce-myparcel'));
+                } else {
+                    echo esc_html(sprintf(__('export_hint_how_many_times', 'woocommerce-myparcel'), count($metaPps)));
```

Comment:
Nu is deze informatie niet meer beschikbaar voor de gebruiker, hoeft dat ook niet?